### ISSUE TITLE
Dagger 2.28 does not have release notes

### ISSUE THREAD

**Issue by utwyko** [google/dagger#1862]
Version has been released two days ago, but does not have release notes.

https://github.com/google/dagger/releases/tag/dagger-2.28

**Comment by danysantiago** [comment]
Yup, we are aware an working on getting the release notes and additional documentation published, bear with us a bit more as you may or may not know, this release is a bit special. :)
